static/app/views/performance/browser/resources/settings.ts

```diff
@@ -1,9 +1,16 @@
 import {t} from 'sentry/locale';
 
-export const MODULE_TITLE = t('Resources');
-export const BASE_URL = 'browser/resources';
-
+export const MODULE_TITLE = t('Assets');
+export const BASE_URL = 'browser/assets'; // Name of the data shown (singular)
 export const MODULE_DESCRIPTION = t(
   'Find large and slow-to-load resources used by your application and understand their impact on page performance.'
 );
+
+export const PERFORMANCE_MODULE_TITLE = t('Resources');
+export const PERFORMANCE_BASE_URL = 'browser/resources';
+export const PERFORMANCE_DATA_TYPE = t('Resource');
+export const PERFORMANCE_MODULE_DESCRIPTION = t(
+  'Find large and slow-to-load resources used by your application and understand their impact on page performance.'
+);
```

Comment:
Prefixed with `Performance` so that once we move to insights, we just delete these and it all makes sense still